src/modules/GridOverlay/Module.js

```diff
@@ -0,0 +1,50 @@
+
+module.exports = function GridOverlay(options,UI) {
+  
+  var output;
+
+  function draw(input, callback, progressObj) {
+
+    progressObj.stop(true);
+    progressObj.overrideFlag = true;
+
+    var step = this;
+    if (!options.step.inBrowser) { // This module is only for browser
```

Comment:
Can we use canvas module of node to get this for node as well?

Reply:
Sorry I can't get you???

Reply:
I mean like as of now we are having this module only for browser but we can make it node compatible by using canvas module from npm as well.
Does that make sense?

Reply:
Oh fine.

Reply:
I will check it out.

Reply:
Sure please!!